src/component/application/ApplicationList.tsx

```diff
@@ -39,6 +23,25 @@ const ClientStrategies = ({ fetchAll, applications }) => {
             : applications;
     }, [applications, filter]);
 
+    const Empty = () => (
```

Comment:
This is now a component inside a component? I would prefer to rename this to `renderNoApplications`.